### PR TITLE
Adds extreme-heat & offsets db

### PR DESCRIPTION
**Pull request by norlandrhagen**

### CODE REVIEW

File: datasets.json [hunk 1]
```diff
@@ -179,5 +179,66 @@
...
+        "href": "https://carbonplan.org/research/offsets-db"
+      },
+      {
+        "label": "Read article",
+        "href": "https://carbonplan.org/research/offsets-db-explainer"
+      }
+    ],
+    "formats": [
+      {
+        "label": "JSON",
```
Comment (andersy005) [inline datasets.json:228]:
```suggestion
        "label": "CSV",
```

File: datasets.json [hunk 2]
```diff
@@ -179,5 +179,66 @@
       "release_date": "05-26-2021",
       "license": "CC BY 4.0"
     }
+  },
+  {
+    "name": "Extreme heat",
```
Comment (katamartin) [inline datasets.json:184]: It looks like this list ~almost chronologically sorted manually.

Maybe worth adding these new entries to the top of the list (OffsetsDB as first element, then extreme heat)?

**Comment by norlandrhagen** [comment]
@katamartin  or @andersy005 how do you normally choose the card colors?

**Comment by katamartin** [comment]
@norlandrhagen I would try to match the color for the article in `research` -- where we try to use colors that roughly map onto topic area (but the logic gets a little squishy sometimes!). So `yellow` for extreme heat and `purple` for OffsetsDB!

**Review by katamartin — APPROVED** [review]

**Pull request merged**